app/diagnostics.go

```diff
@@ -138,16 +137,25 @@ func (a *App) trackActivity() {
 	var incomingWebhooksCount int64
 	var outgoingWebhooksCount int64
 
-	dailyActiveChan := a.Srv.Store.User().AnalyticsActiveCount(DAY_MILLISECONDS)
-	monthlyActiveChan := a.Srv.Store.User().AnalyticsActiveCount(MONTH_MILLISECONDS)
-
-	if r := <-dailyActiveChan; r.Err == nil {
-		activeUsersDailyCount = r.Data.(int64)
-	}
-
-	if r := <-monthlyActiveChan; r.Err == nil {
-		activeUsersMonthlyCount = r.Data.(int64)
-	}
+	activeUsersDailyCountChan := make(chan store.StoreResult, 1)
+	go func() {
+		count, err := a.Srv.Store.User().AnalyticsActiveCount(DAY_MILLISECONDS)
+		if err != nil {
+			mlog.Error(err.Error())
+		}
+		activeUsersDailyCountChan <- store.StoreResult{Data: count, Err: err}
+		close(activeUsersDailyCountChan)
+	}()
+
+	activeUsersMonthlyCountChan := make(chan store.StoreResult, 1)
+	go func() {
+		count, err := a.Srv.Store.User().AnalyticsActiveCount(MONTH_MILLISECONDS)
+		if err != nil {
```

Comment:
We are not login this errors anywhere else, I think is not something expect, I would expect or log everywhere or not log anywhere. so for now, to keep the code as was before, please, remove the `mlog.Error` calls here and in the other async call that you included it.